webui/react/src/hooks/useSettings.ts

```diff
@@ -214,6 +214,7 @@ const useSettings = <T>(config: SettingsConfig, options?: SettingsHookOptions):
   const location = useLocation();
   const storage = useStorage(options?.storagePath || config.storagePath);
   const prevSearch = usePrevious(location.search, undefined);
+  const prevPath = usePrevious(location.pathname, undefined); // console.log(prevSearch, location.search);
```

Comment:
nit: nice to remove console.log

Reply:
> I found an interesting behavior. https://user-images.githubusercontent.com/109113616/195405503-c77349d4-9a29-4d88-9d76-4205ac89f1ad.mov
> 
> question1: `state-related` changes include `column` and `column width` too? question2: as you know, there is a PR to improve `useSettings`. Does the new one have compatibility with this?

@keita-determined 

yes that is an interesting behavior, thanks.

q1: yes
q2: intention is for the other PR to replace this, but I wanted to see if it was possible to get an easy bugfix in the meantime.

Reply:
I dont know if its a good idea, but I think some states should be shared across the projects, such as column and column width, items per page etc. Or we can have `apply to all projects` for these states. Its because I feel weird to have different columns or widths (UI).

Reply:
different columns was the request from design, and I agree with it. different projects you might be interested in different things. if you have different columns you have to have different column widths.

i kinda agree with you on items per page, but I don't think that alone warrants the extra complexity.